test/client/global.js

```diff
@@ -1,9 +1,32 @@
 /* @flow */
 
-import { getHost, getHostName, getPort, getPath, getEnv, getDefaultStageHost, getVersion, getCorrelationID } from '../../src';
+import { PLATFORM, PROTOCOL } from '@paypal/sdk-constants/src';
+
+import { getSDKHost, getHost, getProtocol,
+    getHostName, getPort, getDefaultServiceStageHost,
+    getDefaultAPIStageHost, getStageHost, getFundingEligibility,
+    getAPIStageHost, getDebug, getComponents,
+    getPath, getEnv, getDefaultStageHost,
+    getVersion, getCorrelationID, getPlatform
+} from '../../src';
 
 describe(`globals cases`, () => {
 
+    beforeEach(() => {
```

Comment:
The afterEach() looks good for cleaning up state. Perhaps we get rid of the `beforeEach()` code since that's more test-specific. I would prefer to see the mock values set in the actual test block.